cmd/xk6/main.go

```diff
@@ -139,9 +146,9 @@ func runBuild(ctx context.Context, args []string) error {
 
 func getK6OutputFile() string {
 	if runtime.GOOS == "windows" {
-		return "k6.exe"
+		return ".\\k6.exe"
 	}
-	return "k6"
+	return "./k6"
```

Comment:
I am more inclined to have runDev use a temporary place for the k6 binary instead of overwriting the one in the current directory and then ... possibly deleting it

Reply:
I slightly disagree. I get the overwriting/deletion concern, but this is meant to help with quick dev iterations when run from an extension repo, and it's much easier to have the binary in the CWD than in a temp dir somewhere... If you care about where it's written, you should use `xk6 build --output` instead.

Reply:
But ... you delete the file after that ... unless an env variable is set - so in the majority of the cases you will just delete the file 'k6' in the local directory, nothing else.

Reply:
Right, and if `XK6_SKIP_CLEANUP` is set then I'll have to hunt down the binary in a temp dir, which is more annoying than it overwriting/deleting `./k6`, that should've been produced by a previous `xk6` run anyway.

@na-- Tie breaker? 😄

Reply:
The path is literally a few lines above 
![image](https://user-images.githubusercontent.com/312246/100859778-558e6200-3498-11eb-8611-3e5b988252ca.png)

Reply:
I guess we can resolve this later :)